extensions/resteasy-reactive/quarkus-resteasy-reactive-kotlin/runtime/src/main/kotlin/org/jboss/resteasy/reactive/server/runtime/kotlin/FlowToPublisherHandler.kt

```diff
@@ -27,7 +28,9 @@ class FlowToPublisherHandler : ServerRestHandler {
                 // ensure the proper CL is not lost in dev-mode
                 Thread.currentThread().contextClassLoader = originalTCCL
                 requestContext.result = result.asMulti()
-                requestContext.resume()
+                //run in a direct invocation executor to run the rest of the invocation in the co-route scope
+                //feels a bit fragile, but let's see how it goes
+                requestContext.resume(Executor { it.run() })
```

Comment:
@geoand I needed to do this to make the kotlin tests work. Not sure if I like it much but I think it is probably ok for now.

Reply:
I think we can live with it :)